Library/Homebrew/test/diagnostic_spec.rb

```diff
@@ -41,6 +41,18 @@
     end
   end
 
+  specify "#check_access_lock_dir" do
+    begin
+      mode = HOMEBREW_LOCK_DIR.stat.mode & 0777
+      HOMEBREW_LOCK_DIR.chmod 0555
```

Comment:
Sorry, I mean: what do you think about adding an additional assertation to verify the `HOMEBREW_LOCK_DIR.stat.mode` changes after the `HOMEBREW_LOCK_DIR.chmod 0555`?